eng/mgmt/automation/generation.yml

```diff
@@ -0,0 +1,61 @@
+trigger: none
+
+pr: none
+
+pool:
+  vmImage: 'ubuntu-18.04'
+
+variables:
+  MAVEN_CACHE_FOLDER: $(Pipeline.Workspace)/.m2/repository
+  MAVEN_OPTS: '-Dmaven.repo.local=$(MAVEN_CACHE_FOLDER)'
+  NodeVersion: '14.x'
+
+steps:
+- task: Cache@2
```

Comment:
Is the maven cache that interesting for this pipeline?

Reply:
Maybe I think I could remove the cache, due to generation will usually result to different service with different pom.

Reply:
After testing, I think it still necessary for cache maven dependency for `mvn verify package`.
Actually all services need same cache, so I change this to using key `"maven | ${ OS }"` to do cache.